### DISCUSSION TITLE
augmented image video viewed from the top

### DISCUSSION THREAD

**Discussion by jerodmotley** [sceneview/sceneview#576] [Q&A]
Hi. I'm new Euler and Quaternions but I'm trying to play a video over a detected object. Out of the box, it puts the camera above the video screen instead of directly in front of it. Any help positioning the camera properly would be greatly appreciated.

**Comment by ThomasGorisse** [comment]
When placing video on an augmented image, the video's orientation depends on the anchor pose from the detected image. If the video appears rotated (e.g. only correct from the top), adjust the node's rotation:

```kotlin
AnchorNode(anchor = imageAnchor) {
    VideoNode(
        materialLoader = materialLoader,
        mediaPlayer = mediaPlayer,
        rotation = Rotation(x = -90f, y = 0f, z = 0f) // adjust as needed
    )
}
```

The rotation needed depends on how the image is oriented on the surface (horizontal table vs. vertical wall) and the video's intended viewing angle. Experiment with the X rotation to get the right orientation.
